lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -36,3 +41,16 @@ e2function number entity:isInFaction()
     return this:IsInFaction()
 end
 
+-- Vehicle Entities
+e2function void entity:ejectPodTP(vector pos)
+    pos = clamp(pos)
```

Comment:
I like the idea here, clamping it in-place and continuing to use it, but it's technically bad practice to redefine existing vars.

Perhaps you could rename the arg to `givenPos` and then here you could do `local pos = clamp(givenPos)`

I do that at work sometimes!